tests/unit/CoreQuoteTest.php

```diff
@@ -0,0 +1,328 @@
+		';
+	}
+
+	/**
+	 * Test case for retrieving core quote block fields and attributes.
+	 *
+	 * Covers : 'citation', 'className', 'cssClassName' and 'value'.
+	 */
+	public function test_retrieve_core_quote_fields_and_attributes(): void {
+		$block_content = '
+			<!-- wp:quote {"className":"custom-quote-class"} -->
+			<blockquote class="wp-block-quote custom-quote-class"><p>This is a sample quote block content.</p><cite>Author Name</cite></blockquote>
+			<!-- /wp:quote -->
+		';
+
+		// Update the post content with the block content.
+		wp_update_post(
+			[
+				'ID'           => $this->post_id,
+				'post_content' => $block_content,
+			]
+		);
+
+		$query     = $this->query();
+		$variables = [
+			'id' => $this->post_id,
+		];
+
+		$actual = graphql( compact( 'query', 'variables' ) );
+		error_log( print_r( $actual, true ) );
+
+		$this->assertArrayNotHasKey( 'errors', $actual, 'There should not be any errors' );
+		$this->assertArrayHasKey( 'data', $actual, 'The data key should be present' );
+		$this->assertArrayHasKey( 'post', $actual['data'], 'The post key should be present' );
+
+		$this->assertEquals( $this->post_id, $actual['data']['post']['databaseId'], 'The post ID should match' );
+
+		$this->assertEquals( 1, count( $actual['data']['post']['editorBlocks'] ), 'There should be only one block' );
+
+		$block = $actual['data']['post']['editorBlocks'][0];
+
+		$this->assertNotEmpty( $block['apiVersion'], 'The apiVersion should be present' );
+		$this->assertEquals( 'text', $block['blockEditorCategoryName'], 'The blockEditorCategoryName should be text' );
+		$this->assertNotEmpty( $block['clientId'], 'The clientId should be present' );
+
+		$this->assertEquals( $block['cssClassNames'][0], 'custom-quote-class' );
+
+		$this->assertEmpty( $block['innerBlocks'], 'There should be no inner blocks' );
+		$this->assertEquals( 'core/quote', $block['name'], 'The block name should be core/quote' );
+		$this->assertEmpty( $block['parentClientId'], 'There should be no parentClientId' );
+		$this->assertNotEmpty( $block['renderedHtml'], 'The renderedHtml should be present' );
+
+		// Verify the attributes..
+
+		// WordPress 6.4+ adds layout styles, so `cssClassName` needs to be checked separately.
+		$this->assertStringContainsString( 'wp-block-quote', $block['attributes']['cssClassName'] );
+		$this->assertStringContainsString( 'custom-quote-class', $block['attributes']['cssClassName'] );
+		unset( $block['attributes']['cssClassName'] );
+
+		$this->assertEquals(
+			[
+				'anchor'          => null,
+				'backgroundColor' => null,
+				'citation'        => 'Author Name',
+				'className'       => 'custom-quote-class',
+				'fontFamily'      => null,
+				'fontSize'        => null,
+				'gradient'        => null,
+				'lock'            => null,
+				'style'           => null,
+				'textColor'       => null,
+				'value'           => '<p>This is a sample quote block content.</p>',
+				'layout'          => null,
+				'textAlign'       => null,
```

Comment:
You removed these from the query entirely, so they should be _removed_ here, not exiting with a value of null (ref: https://github.com/rtCamp/wp-graphql-content-blocks/actions/runs/11011696682/job/30576344056?pr=23#step:8:282)

Reply:
yupp, i justt figured that out 😓

Reply:
no need to comment them out, just _remove_ them.